about.qmd

```diff
@@ -51,17 +51,19 @@ If you would like to see a blog post on a particular topic, submit an issue on o
 - [X] [Using Squidpy with AtoMx&#8482; SIP exports](posts/squidpy-essentials/squidpy-essentials.qmd)
 - [X] [QC and normalization of RNA data](posts/normalization/index.qmd)
 - [X] [FOV QC from single-cell gene expression in spatial dataset](posts/fov-qc/index.qmd)
+- [X] [Impacts of background in CosMx<sup>&#174;</sup> data](posts/background/index.qmd)
 - [ ] Batch correction 
 - [X] [The spatial algorithms zoo: recommended algorithms and efficient code](posts/spatial-algorithm-zoo/index.qmd)
 - [ ] A generally satisfying set of UMAP parameters for CosMx data
 - [X] [The impact of segmentation error on differential expression analyses](posts/segmentation-error/index.qmd)
 - [ ] Quick & comprehensive searches for interesting trends with "Everything vs. everything DE"
-- [ ] Smoothing single cell gene expression for enhanced plotting
+- [X] [Smoothing single cell gene expression for enhanced plotting](posts/marker-gene-smoothing/index.qmd)
```

Comment:
I feel this needs a better name, it's not just about the plotting but also has influence on cell typing.

Reply:
Removed from this section